package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "legalize",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I wouldn't say that this one is actually a bug fix. Neither a feature.

From the use case of being concatenated into a 3rd party library, it would be safer to hookup Legalize onto the window object no mather what. I checked the behavior of jQuery, just to be sure, that they approach the requirejs thingy the same way.